### PR TITLE
For Oracle ODB@A Resources Add annotation about ForceNew to docs

### PR DESCRIPTION
**Pull request by eelhomsi**
## Community Note

* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
* Please do not leave comments along the lines of "+1", "me too" or "any updates", they generate extra noise for PR followers and do not help prioritize for review

## Description

Add some additional information explaining undateable fields and their behaviour.

## PR Checklist

- [X] I have followed the guidelines in our [Contributing Documentation](../blob/main/contributing/README.md).
- [X] I have checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change.
- [X] I have checked if my changes close any open issues. If so please include appropriate [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) below.
- [X] I have updated/added Documentation as required written in a helpful and kind way to assist users that may be unfamiliar with the resource / data source.
- [X] I have used a meaningful PR title to help maintainers and other users understand this change and help prevent duplicate work. 
For example: “`resource_name_here` - description of change e.g. adding property `new_property_name_here`”

## Change Log

Update documentation for OracleDatabase resources detailing updatable fields.

This is a (please select all that apply):

- [ ] Bug Fix
- [ ] New Feature (ie adding a service, resource, or data source)
- [X] Enhancement
- [ ] Breaking Change

> [!NOTE] 
> If this PR changes meaningfully during the course of review please update the title and description as required.

### CODE REVIEW

**Review by jackofallops — APPROVED** [review]
Thanks @eelhomsi - This LGTM 👍

File: website/docs/r/oracle_cloud_vm_cluster.html.markdown [hunk 1]
```diff
@@ -81,73 +81,75 @@ resource "azurerm_oracle_cloud_vm_cluster" "example" {
...
+* `display_name` - (Required) The user-friendly name for the Cloud VM Cluster. Changing this forces a new Cloud VM Cluster to be created. The name does not need to be unique.
 
-* `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version.
+* `gi_version` - (Required) A valid Oracle Grid Infrastructure (GI) software version. Changing this forces a new Cloud VM Cluster to be created.
 
-* `hostname` - (Required) The hostname for the Cloud VM Cluster without suffix.
+* `hostname` - (Required) The hostname for the Cloud VM Cluster without suffix. Changing this forces a new Cloud VM Cluster to be created.
 
-* `license_model` - (Required) The Oracle license model that applies to the Cloud VM Cluster, either `BringYourOwnLicense` or `LicenseIncluded`.
+* `hostname_actual` - The hostname for the Cloud VM Cluster with suffix. Changing this forces a new Cloud VM Cluster to be created.
```
Comment (jackofallops) [inline website/docs/r/oracle_cloud_vm_cluster.html.markdown:97]: looks like this is already covered in the Attributes reference below as it's `Computed` only property?
```suggestion
```

**Review by jackofallops — CHANGES_REQUESTED** [review]
Hi @eelhomsi - Thanks for resolving the conflicts.  I'm not sure if I missed it last time, or if it crept in via the rebase, but there's just one change needed below and then we're good to go.

File: website/docs/r/oracle_cloud_vm_cluster.html.markdown (hunk 1 above)
Comment (eelhomsi) [reply website/docs/r/oracle_cloud_vm_cluster.html.markdown:97]: Good point, applied your suggestion thanks!

**Pull request merged**